src/plugins/intel_cpu/src/nodes/conv.cpp

```diff
@@ -1385,8 +1385,7 @@ void Convolution::prepareParams() {
                                      const dnnl::primitive_attr attr) -> dnnl::primitive_desc {
             dnnl::memory::desc dnnlBiasDesc;
             if (biasDescPtr) {
-                // WA to align IR bias representation (3 to 5 rank tensors) to oneDNN representation (1 rank tensor)
-                dnnlBiasDesc = biasDescPtr->getDnnlDesc().reshape({dstDesc.get_dims()[1]});
+                dnnlBiasDesc = biasDescPtr->getDnnlDesc();
             }
```

Comment:
Since the bias node is const node, the reshape would be implemented in the period of cpu graph compiling period. And reshape execution would be in place and only memory descriptor changed. No runtime overhead in my understanding.